### DISCUSSION TITLE
Custom Slider

### DISCUSSION THREAD

**Discussion by GaborTheHutt** [kikipoulet/SukiUI#214] [Q&A]
Hi,

I try to style sliders similar to the iOS control center.

But when I try to custom style the slider Suki UI seems to override my style.
Can someone please explain to me how to implement custom styles correctly.

![image](https://github.com/kikipoulet/SukiUI/assets/60019588/d1c0ee52-6385-4a77-b39e-e0ec5a4bf422)

**Comment by kikipoulet** [comment]
Be sure that you add your Slider styles after adding SukiUI library in your App.axaml
